packages/rfc6287/rfc6287.1.0.3/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+version:      "1.0.3"
+name:         "rfc6287"
+homepage:     "https://github.com/sg2342/ocaml-rfc6287"
+dev-repo:     "https://github.com/sg2342/ocaml-rfc6287.git"
+bug-reports:  "https://github.com/sg2342/ocaml-rfc6287/issues"
+author:       ["Stefan Grundmann <sg2342@googlemail.com>"]
+maintainer:   ["Stefan Grundmann <sg2342@googlemail.com>"]
+license:      "ICS"
```

Comment:
this looks wrong, is it `ISC`?  the repository contains a BSD 2 clause license, so maybe `BSD2` is more appropriate.